BrainPortal/lib/cbrain_task_generators/templates/task_params.html.erb.erb

```diff
@@ -108,7 +108,7 @@
   <ul class="tsk-prm-list">
     <li>
       <%%
-        input.(id, name,
+        input.(id, name+"[]",
```

Comment:
Yes I know @prioux, this is precisely where the bug was coming from (arrays have to have a name ending with []). Do you actually have a concern about the code or is it just a general note?